mava/networks.py

```diff
@@ -81,7 +81,7 @@ def __call__(self, observation: chex.Array) -> chex.Array:
             x = self.activation_fn(x)
 
         # Reshape should keep the batch and agent dimensions unchanged.
-        return x.reshape((x.shape[0], x.shape[1], -1))
+        return jax.lax.collapse(x, -3)
```

Comment:
I'd say remove the comment above for these two
```suggestion
        # Collapse (merge) the last three dimensions (width, height, channels)
        # Leave the batch, agent and time (if recurrent) dims unchanged.
        return jax.lax.collapse(x, -3)
```